### ISSUE TITLE
WCAG 2.2AA Compliance Check

### ISSUE THREAD

**Issue by stefanikauppila** [Fractal-Learning/fractal-learning-web-app#78]
We need to provide our users with Policy, Terms of Use, Data Security Information, etc. and Accessibility is one area we need to understand. Ask the system to analyze its own WCAG 2.2 AA alignments to provide areas of compliance and areas of non-compliance and how to fix/improve to create compliance. This will help us create an Accessibility Compliance chart.

**Comment by Nfinley** [comment]
A few things to check: 
* Screen reader is working correctly on all pages
* Tabbing is working
* No focus traps
* App navigation

Run the AXE web browser plugin
